test/sql/copy/csv/csv_duck_fuzz.test

```diff
@@ -0,0 +1,26 @@
+# name: test/sql/copy/csv/csv_duck_fuzz.test
+# description: Issues found from DuckFuzz regarding the CSV Reader
+# group: [csv]
+
+statement ok
+PRAGMA enable_verification
+
+foreach paramenter sep delim quote new_line escape nullstr columns auto_type_candidates header auto_detect sample_size all_varchar dateformat timestampformat normalize_names compression skip max_line_size maximum_line_size ignore_errors store_rejects rejects_table rejects_scan rejects_limit force_not_null buffer_size decimal_separator parallel null_padding allow_quoted_nulls column_types dtypes types names column_names comment encoding rfc_4180
+
+statement error
+SELECT NULL FROM sniff_csv(NULL, ${paramenter} := NULL)
+----
+
```

Comment:
can you also add what kind of error is expected? Like in https://github.com/duckdb/duckdb/blob/adc6f607a71b87da2d0a7550e90db623e9bea637/test/sql/pivot/pivot_in_subquery.test#L63